lib/findProjectRoot.js

```diff
@@ -11,7 +12,12 @@ function findRecursive(directory) {
   const pathToPackageJson = path.join(directory, 'package.json');
 
   if (fs.existsSync(pathToPackageJson)) {
-    return directory;
+    const packageDescription = FileUtils.readJsonFile(pathToPackageJson);
+    const { isRoot = true } = packageDescription['import-js'] || {};
+
+    if (isRoot) {
+      return directory;
+    }
```

Comment:
This looks useful, and I'm assuming it's a continuation of https://github.com/Galooshi/import-js/pull/462? An update to `findProjectRoot-test.js` as well as an entry in the README and I think this is good to go. 👍